### ISSUE TITLE
Results on Diving48 Datasets without pretrained weights

### ISSUE THREAD

**Issue by DAVEISHAN** [vt-vl-lab/SDN#7]
Hi Choi, 

Very interesting paper! Thanks for sharing the repo too!

I am trying to do action classification on the Diving48 dataset with **3D-ResNet 18 layers** without using any pre-trained weights. I am getting a very poor result: **9.5% classification accuracy.** Do you have any suggestions on improving this number or is it correct? Can you pl give me any reference paper which reports 3D CNN results without using any pre-trained weights?

My train clips: 112 x 112 resolution, and 16 frames 
I do testing only on a single 16 frames clip for a video.

Hope to hear from you soon.

-Ishan

**Comment by jinwchoi** [comment]
Hi @DAVEISHAN 

I haven't tried to run 3D-ResNet-18 from scratch on the Diving48 dataset and I haven't seen any paper trains a model from scratch on Diving48. But I guess 9.5% might be quite reasonable, as Kinetics pre-training can only achieve ~18%.

**Comment by DAVEISHAN** [comment]
Thank you @jinwchoi ! I also wanted to know the inference strategy for Diving48: Do you take only a single clip for a video prediction or average of multiple clip predictions?(for eg people take 10 clip prediction per video to get a prediction for a UCF101 video.) 

-Ishan
